lms/views/dashboard/api/course.py

```diff
@@ -44,12 +50,6 @@ def __init__(self, request) -> None:
         self.dashboard_service = request.find_service(name="dashboard")
         self.assignment_service = request.find_service(name="assignment")
 
-        self.admin_organizations = (
```

Comment:
This is now on the constructor, which is not that bad for very simple things but adding extra logic complicates testing etc.

Moving it to a @property at the bottom.